src/include/rbd/librbd.h

```diff
@@ -587,6 +587,14 @@ CEPH_RBD_API int rbd_mirror_image_instance_id_list(rados_ioctx_t io_ctx,
 CEPH_RBD_API void rbd_mirror_image_instance_id_list_cleanup(char **image_ids,
                                                             char **instance_ids,
                                                             size_t len);
+CEPH_RBD_API int rbd_mirror_image_info_list(
+    rados_ioctx_t io_ctx, rbd_mirror_image_mode_t *mode_filter,
+    const char *start_id, size_t max, char **image_ids,
```

Comment:
Nit: would it make sense to consolidate `max` and `num_entries` to a single in/out parameter?

Reply:
@dillaman This change was from #32434 this PR was rebased on, because at that moment #32434 was not still merged. Now as #32434 is merged this change is in the master already.

I suppose we could  consolidate `max` and `num_entries` as an additional PR, though note, that all other `rbd_mirror_image_*_list` functions have both params, so it would be inconsistent.

Reply:
No worries, I forgot it was part of that other PR.